.github/workflows/ansible-test.yml

```diff
@@ -86,36 +87,76 @@ jobs:
           - 3.7
           - 3.8
           - 3.9
+        vault:
+          - 1.6.0
+          - 1.5.5
+          - 1.4.7
         exclude:
           - ansible: stable-2.9
             python: 3.9
 
     steps:
+      - name: Initialize env vars
+        run: |
+          COLLECTION_PATH=ansible_collections/${NAMESPACE}/${COLLECTION_NAME}
+          LOOKUP_HASHI_VAULT_PATH=${COLLECTION_PATH}/tests/integration/targets/lookup_hashi_vault
```

Comment:
It would make sense to move the setup part to a different target `setup_hashi_vault`, so that once you add more modules / plugins you can simply depend on `setup_hashi_vault` as well. (But you can also do that split-up in a later PR.)

Reply:
I totally agree, but it will later/in a different PR. There's more I'd like to do as well, like not just being able to set a vault version, but also supply the address of a running instance, in which case download/extract/set policies and secrets, etc. can all be skipped.

But yeah separating it out later will let the whole thing get leaner. I just didn't want to add even more to what are already a lot of changes.